src/components/ChatEntry.js

```diff
@@ -1,22 +1,42 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import { DateTime } from 'luxon';
+import TimeStamp from './TimeStamp';
+import { useState } from 'react';
 
 const ChatEntry = (props) => {
+
+  const [isLiked, setIsLiked] = useState(props.isLiked);
+  const heartButton = isLiked ? '❤️':'🤍'
+
+  const toggleLike = () => {
+      setIsLiked(!isLiked);
+      props.updateLike(props.id);
+  }
+
+
   return (
     <div className="chat-entry local">
```

Comment:
This might be a good place to determine if the chat entry is "local" or "remote," which can determine how it appears (aka a different CSS class besides `local`)